zephyr/Kconfig

```diff
@@ -67,4 +67,10 @@ config CROSS_CORE_STREAM
 	  can be processed on different cores, however, each stream
 	  is processed entirely on single core.
 
+config SOF_BOOT_TEST
+       bool "enable SOF run-time testing"
+       depends on ZTEST
+       help
```

Comment:
Can we expand this more and tell the users that its for testing use only and should not be enabled for normal audio usage. i.e what this enables/disables and impact.